src/test/ui/nll/closure-requirements/escape-argument-callee.stderr

```diff
@@ -9,6 +9,12 @@ error: free region `ReFree(DefId(0/1:9 ~ escape_argument_callee[317d]::test[0]::
    |
 LL |         let mut closure = expect_sig(|p, y| *p = y);
    |                                             ^^^^^^
+   |
+   = note: because of an outlives relation created at `bb0[0]`
+           because of an outlives relation created at `bb0[1]`
+           because of an outlives relation created at `bb0[1]`
+           because of an outlives relation created at `bb0[0]`
+           because `'_#3r` is universally quantified
```

Comment:
didn't we remove this output? I guess this PR is independent from that one?

Reply:
Yes, the other PR is not in master yet.